src/store/sliceCategory/selectorsCategory.ts

```diff
@@ -0,0 +1,38 @@
+import { LOAD_STATUSES } from "../../constants";
+import { rootStore } from "../store";
+import { State } from "./sliceCategory";
+import { getCategories } from "store/sliceCategories/selectorsCategories";
+
+export const getCategorySlice = (state: rootStore): State => state.category;
+export const getLoadStatusSlice = (state: rootStore): LOAD_STATUSES =>
+  getCategorySlice(state).loadStatus;
+  
+  export const getCategory = (state: rootStore) => getCategorySlice(state).category;
+  export const getIsLoadingSeletor = (state:rootStore) =>  {
+    return getLoadStatusSlice(state) === LOAD_STATUSES.LOADING;
+  }
+   export const getIsLoadedSeletor = (state:rootStore) =>  {
```

Comment:
```suggestion
   export const getIsLoadedSeletor = (state:rootStore): boolean=>  {
```
и в других местах тоже